### DISCUSSION TITLE
stuck with 4.6.1

### DISCUSSION THREAD

**Discussion by HarshilPatel007** [prisma/prisma#19109] [Q&A]
### Bug description

I'm planning to utilize Prisma for my web app (nestjs) that has multi-tenant db architecture. I've done that successfully. But, I'm stuck with `v4.6.1` of `prisma` and `prisma-client`. Actually, the app works fine with the `v4.6.1`. But, as soon as I bump the version, I get an extremely slow response and in the end got errors that result in a failed response (500). That doesn't happen with the `v4.6.1`. All things work perfectly and smoothly without any hurdles with that version.
Here is the test repo. If anyone wants to see the code. : https://github.com/HarshilPatel007/my-nest-app/tree/prisma-test

Here is the error I got.
```
[Nest] 15572  - 2023-05-04, 11:07:22 a.m.   ERROR [ExceptionsHandler] Invalid string length
RangeError: Invalid string length
    at join (node:internal/util:384:20)
    at reduceToSingleString (node:internal/util/inspect:1947:10)
    at formatRaw (node:internal/util/inspect:1063:15)
    at formatValue (node:internal/util/inspect:817:10)
    at inspect (node:internal/util/inspect:347:10)
    at formatWithOptionsInternal (node:internal/util/inspect:2167:40)
    at formatWithOptions (node:internal/util/inspect:2029:10)
    at console.value (node:internal/console/constructor:324:14)
    at console.log (node:internal/console/constructor:360:61)
    at UserService.getUserByEmail (D:\nestjs\my-app\src\user\user.service.ts:40:13)
```

### How to reproduce
- The repo contains version `4.6.1`. To reproduce the bug, change it to `4.7.1` or higher.
1. clone the repo. I've given
2. set-up the mongoDB (create a database named **DEFAULT** with the collection name **EMPTY_COLLECTION**) and add your credentials in `.env` file
3. run the `npm run prisma-all` command to set up the table in the database.
4. now, `npm run start:dev`
5. try to create a user, `http://localhost:3000/user/create/` (POST). add header `dbnm : default`
```json
 {
    "email": "{{$randomEmail}}",
    "username": "{{$randomUserName}}",
    "password": "Pass@123#123",
    "fullame": "{{$randomFullName}}",
    "age": 23
}
```

### Expected behavior

- The query should work without any errors and with reasonable time.

### Prisma information

- Please refer to the repo. I've mentioned.

### Environment & setup

- OS: Windows
- Database: MongoDB 
- Node.js version: v16.16.0

### Prisma Version

```
Environment variables loaded from .env
prisma                  : 4.7.1
@prisma/client          : 4.7.1
Current platform        : windows
Query Engine (Node-API) : libquery-engine 272861e07ab64f234d3ffc4094e32bd61775599c (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Migration Engine        : migration-engine-cli 272861e07ab64f234d3ffc4094e32bd61775599c (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine    : introspection-core 272861e07ab64f234d3ffc4094e32bd61775599c (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary           : prisma-fmt 272861e07ab64f234d3ffc4094e32bd61775599c (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Format Wasm             : @prisma/prisma-fmt-wasm 4.7.1-1.272861e07ab64f234d3ffc4094e32bd61775599c
Default Engines Hash    : 272861e07ab64f234d3ffc4094e32bd61775599c
Studio                  : 0.477.0

```

**Comment by nurul3101** [comment]
Hello @HarshilPatel007 👋 

Thanks for sharing the reproduction repository, I'll clone it and try reproducing the error.
Meanwhile, to confirm you get this error on all Prisma versions after 4.6.1, right? So, if you use the latest prisma version `4.13.0` then also you get this error?

**Reply by HarshilPatel007** [reply]
@nurul3101 , Thanks for showing intrest into it. Yes, This only works with the 4.6.1 only. Not on the version greater than 4.6.1.
Please tell me if you need any help to set up the project.
I'll update the Postman Collection and env-example so that you can run it.

**Update**: env-example and Postman Collection pushed to the branch `prisma-test`.

**Reply by HarshilPatel007** [reply]
Hi @nurul3101 , is there any updates?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
